apis/r/src/riterator.cpp

```diff
@@ -27,17 +27,16 @@ namespace tdbs = tiledbsoma;
 //'   \item{\code{sr_next}}{returns the next chunk}
 //' }
 //'
-//' @param ctx An external pointer to a TileDB Context object
 //' @param uri Character value with URI path to a SOMA data set
+//' @param config Named chracter vector with \sQuote{key} and \sQuote{value} pairs
+//' used as TileDB config parameters. If unset default configuration is used.
```

Comment:
Suggest to remove "If unset default configuration is used."

Run compileAttributes() and roxygenize() and I think you can [ci skip] commit this.

Reply:
done ✨